apps/account-api/src/services/keys.service.ts

```diff
@@ -71,9 +74,7 @@ export class KeysService {
         ],
       };
 
-      const encodedPayload = u8aToHex(
-        u8aWrapBytes(this.blockchainService.createItemizedSignaturePayloadV2Type(payload).toU8a()),
-      );
+      const encodedPayload = u8aToHex(this.blockchainService.createItemizedSignaturePayloadV2Type(payload).toU8a());
```

Comment:
This is not an extrinsic; it's a payload to be included in an extrinsic call. So don't we want to byte-wrap?

Reply:
It is the same argument I mentioned before. It depends how it is going to be signed.

To get rid of confusion I have in mind to return different types for raw and wrapped payloads to the consumer would know this is wrapped or not for ease of use.